acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -107,11 +107,14 @@ func TestAPIGateway_Basic(t *testing.T) {
 			logger.Log(t, "creating static-client pod")
 			k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/bases/static-client")
 
+			k8s.RunKubectl(t, ctx.KubectlOptions(t), "wait", "--for=condition=available", "--timeout=5m", fmt.Sprintf("deploy/%s", "static-server"))
```

Comment:
If at the end of the 5m timeout the pods still aren't available, will the test continue on or fail?

Reply:
The test would just continue, it will fail further down the line when other conditions are not met.